### DISCUSSION TITLE
Organizations - Self Management

### DISCUSSION THREAD

**Discussion by craschka** [keycloak/keycloak#31758] [Q&A]
Hi, 
Is there a possibility that a user of an organization becomes an "organization admin", i.e. he can add further users and assign rights, but not in the whole realm?

**Comment by darius-m** [comment]
The purpose of realms is, in theory, to emulate organizations in the SSO configuration space. What would the "organization admin" be able to see in your scenario (i.e., what would the conditions for managing users be, outside of realms)? I think fine-grained permissions would be the best lead, but I'm not sure whether it would provide the required functionality.

**Reply by craschka** [reply]
It's about the new feature from v25 (https://www.keycloak.org/docs/latest/server_admin/index.html#_managing_organizations). 

You can create different organizations within a realm, but they all share the same realm settings (clients etc.) (SaaS solution). (The users within an organization should neither be able to see users outside (but in the same realm) nor make changes to them.

**Reply by darius-m** [reply]
Ah, my bad, it seems I have missed that feature. I'll also look into it, thanks for correcting me.

**Reply by craschka** [reply]
Are there any news on this topic? Is this something the team is working on?
